src/PromoSection/index.js

```diff
@@ -1,36 +1,53 @@
+/*
+ * React fn component `PromoSection`
+ * FIXME: Similar to ../Subscribe and ../TrySection
```

Comment:
I found that some components are very similar and could probably be merged (using code such as Props for the subtle differences). Is such a refactoring desirable, or is it common practice in React to just copy-paste components (some times exactly the same) @iAdramelk? Thanks

Reply:
def not a common practice to copy stuff blindly. Hard to say anything about this case specifically. Like in any language it's a balance of time, effort, need to generalize, etc.

Please, don't use FIXME, TODO, etc - if see a thing that should be fixed but in the scope of this PR, create a ticket instead.

Reply:
Sure. I don't think it was done blindly and [React docs](https://reactjs.org/docs/rendering-elements.html) do state that "React elements are ... cheap to create" so maybe people tend interpret that as copy-paste a lot is OK?

Anyway, removed all code tags and opened #777 (lucky number 🍀).